plotly/offline/offline.py

```diff
@@ -62,7 +62,9 @@ def get_image_download_script(caller):
     """
 
     if caller == 'iplot':
-        check_start = 'if(document.readyState == \'complete\') {{'
+        # note: 5500/5.5s is a handpicked delay. The script will not run again
+        # once this time has elapsed after the timestamp is instantiated.
+        check_start = 'var d=(new Date()).getTime();if(!(d>{utime}+5500)){{'
```

Comment:
Man, sorry I keep dragging you through this, but `var d = ...` is the same as `window.d = ...`. If anything I think it's just a little more confusing. Can we do:

`check_start = 'if(!(new Date()).getTime()>{utime}+5500){{'`